controllers/masteruserrecord/masteruserrecord_controller_test.go

```diff
@@ -675,57 +877,44 @@ func TestModifyUserAccounts(t *testing.T) {
 func TestSyncMurStatusWithUserAccountStatuses(t *testing.T) {
 	logf.SetLogger(zap.New(zap.UseDevMode(true)))
 	s := apiScheme(t)
+	spaceBinding := spacebindingtest.NewSpaceBinding("john", "john-space", "admin", "john-123")
+	space := spacetest.NewSpace(commontest.HostOperatorNs, "john-space",
+		spacetest.WithLabel(toolchainv1alpha1.SpaceCreatorLabelKey, "john-123"),
+		spacetest.WithSpecTargetCluster(commontest.MemberClusterName))
 
 	t.Run("mur status synced with updated user account statuses", func(t *testing.T) {
 		// given
 		// setup MUR that wil contain UserAccountStatusEmbedded fields for UserAccounts from commontest.Member2ClusterName and "member3-cluster" but will miss from commontest.MemberClusterName
 		// then the reconcile should add the misssing UserAccountStatusEmbedded for the missing commontest.MemberClusterName cluster without updating anything else
 		mur := murtest.NewMasterUserRecord(t, "john",
+			murtest.WithOwnerLabel("john-123"),
 			murtest.Finalizer("finalizer.toolchain.dev.openshift.com"),
 			murtest.StatusCondition(toBeNotReady(toolchainv1alpha1.MasterUserRecordProvisioningReason, "")),
-			murtest.AdditionalAccounts(commontest.Member2ClusterName, "member3-cluster"))
+			murtest.StatusUserAccount(commontest.Member2ClusterName, toBeProvisioned()))
 
 		userAccount := uatest.NewUserAccountFromMur(mur,
 			uatest.StatusCondition(toBeNotReady("Provisioning", "")), uatest.ResourceVersion("123abc"))
 		userAccount2 := uatest.NewUserAccountFromMur(mur,
-			uatest.StatusCondition(toBeNotReady("Provisioning", "")), uatest.ResourceVersion("123abc"))
-		userAccount3 := uatest.NewUserAccountFromMur(mur,
-			uatest.StatusCondition(toBeNotReady("Provisioning", "")), uatest.ResourceVersion("123abc"))
-
-		mur.Status.UserAccounts = []toolchainv1alpha1.UserAccountStatusEmbedded{
-			{
-				Cluster: toolchainv1alpha1.Cluster{
-					Name: commontest.Member2ClusterName,
-				},
-				UserAccountStatus: userAccount2.Status,
-			},
-			{
-				Cluster: toolchainv1alpha1.Cluster{
-					Name: "member3-cluster",
-				},
-				UserAccountStatus: userAccount3.Status,
-			},
-		}
+			uatest.DeletedUa(),
+			uatest.WithFinalizer(),
+			uatest.StatusCondition(toBeNotReady("Terminating", "")), uatest.ResourceVersion("123abc"))
```

Comment:
```suggestion
			uatest.StatusCondition(toBeNotReady(toolchainv1alpha1.UserAccountTerminatingReason, "")), uatest.ResourceVersion("123abc"))
```

Reply:
same as for the other cases - we don't have to use constants at all places in unit tests. Mainly when the logic shouldn't care what the actual value of the reason is.